src/appliance-onboarding-script/appliance_setup/run.py

```diff
@@ -103,6 +103,11 @@ def _populate_default_values_of_optional_fields_in_config(config):
     except IndexError:
         log_level = "INFO"
 
+    try:
```

Comment:
Can we document the usage of the new parameters introduced somewhere? It could be a comment block or readme

Reply:
Added the comment block.